lib/reporters/cli/cli-utils.js

```diff
@@ -75,6 +75,17 @@ cliUtils = {
         'right-right': '-'
     },
 
+    /**
+     * A helper function which returns a string with the timestamp of current time
+     * (i.e The time when this function is called)
+     *
+     * @returns {String} - A string carrying timestamp
+     */
+    currentTime: function () {
+        return new Date().toString();
```

Comment:
I feel [toLocaleString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString) works best here.

Check if there's any other helpful instance method available in the Date's [prototype](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date).

Reply:
We can move this method to the CLI reporter itself if it's not used in multiple places.
Better have fewer utils. 😅

Reply:
> I feel [toLocaleString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString) works best here.
> 
> Check if there's any other helpful instance method available in the Date's [prototype](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date).

So, can we use `toUTCString` and `toLocaleString`. I am of the opinion to allow an option to show UTC time, so that users with newman instances in different timezones can use it out of the box. The flags can be `--reporter-cli-show-timestamp` and `--reporter-cli-show-timestamp-utc` or something similar but maybe shorter. 

What do you think?